src/services/admin.js

```diff
@@ -33,8 +40,47 @@ module.exports = class AdminHelper {
 			}
 			let result = {}
 
-			const mentor = await mentorQueries.getMentorExtension(userId)
-			const isMentor = mentor !== null
+			const getUserDetails = await menteeQueries.getUsersByUserIds([userId])
+			if (getUserDetails.length <= 0) {
+				return responses.failureResponse({
+					statusCode: httpStatusCode.bad_request,
+					message: 'USER_NOT_FOUND',
+					result,
+				})
+			}
+			const userInfo = getUserDetails[0]
+			const isMentor = userInfo.isMentor == true ? true : false
+			const isAlreadyUnderDeletion = userInfo.status === common.UNDER_DELETION_STATUS
+
+			const getUserDetailById = await userRequests.fetchUserDetails({ userId })
+			const roleTitles = getUserDetailById.data.result.user_roles.map((u) => u.title)
+			const isSessionManager = roleTitles.includes(common.SESSION_MANAGER_ROLE)
```

Comment:
@sumanvpacewisdom remove or comment out the logic for  session manager delete flow